pymc3/backends/base.py

```diff
@@ -256,6 +256,17 @@ class MultiTrace:
     For any methods that require a single trace (e.g., taking the length
     of the MultiTrace instance, which returns the number of draws), the
     trace with the highest chain number is always used.
+
+    Attributes
+    ----------
+        nchains : int
+            Number of chains in the `MultiTrace`.
+        chains : `List[int]`
+            List of chain indices
+        report : str
+            Report on the sampling process.
+        varnames : `List[str]`
```

Comment:
Same as the above, not sure if type hints should be rendered as code or simply emphasized

```suggestion
        varnames : ``List[str]``
```